boards/xtensa/intel_s1000_crb/doc/intel_s1000.rst

```diff
@@ -218,10 +218,98 @@ However, we need to short 2 pins on Host Connector J3 via a 3.3k resistor
 (simple shorting without the resistor will also do) for debugging to work.
 Those 2 pins are Pin5 HOST_RST_N_LT_R) and Pin21 (+V_HOST_3P3_1P8).
 
+MCUbootloader Support
+=====================
+
+Intel S1000 on Zephyr supports the open source, cross-RTOS MCUbootloader.
```

Comment:
replace MCUbootloader  with MCUBoot Bootloader

Reply:
Also, it's Zephyr on Intel S1000, not Intel S1000 on Zephyr.

Reply:
Corrected.